### PR TITLE
feat(orc-1289): add new fields to profile data capture node

### PR DESCRIPTION
**Pull request by acMaiaOnf**
# Problem
New profile data capture node has configurable values and we want to show/hide data capture fields

# Solution

Set new data fields and required values based on the config from the profile capture node

## Checklist

_put `n/a` if item is not relevant to PR changes_

- [ ] Has the CHANGELOG been updated?
- [ ] Has the README been updated?
- [ ] Has the CONTRIBUTING doc been updated?
- [ ] Has the RELEASE_GUIDELINES been updated?
- [ ] Has the TESTING_GUIDELINES been updated?
- [ ] Has the MIGRATION doc been updated for any MAJOR breaking changes?
- [ ] Has the MIGRATION doc been updated for any MINOR breaking changes, including any translation strings or keys changes?
- [ ] Have any new automated tests been implemented or the existing ones changed?
- [ ] Have any new manual tests been written down or the existing ones changed?
- [ ] Have any new strings been translated or the existing ones changed?

### CODE REVIEW

File: src/components/Capture/PhoneNumberInput/index.tsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  onChange: (ev: {
+    target: {
+      value: string
+    }
+  }) => void
+  smsNumberCountryCode?: Country
+  options: SdkOptions
+} & WithLocalisedProps
+
+const PhoneNumberInput = ({
```
[the remark this answers is not in the document]
Comment (acMaiaOnf) [reply src/components/Capture/PhoneNumberInput/index.tsx:37]: I tried that at first, but I was having more issues with that. I couldn't really get it work properly.
But we also don't need to duplicate it. I think I'll create profile data phone number input with the same lazy loading and styling, but with the new design.

[the remark this answers is not in the document]
Comment (josokinas) [reply src/components/Capture/PhoneNumberInput/index.tsx:37]: Or at least include whatever can be exported/imported from that original component, to reduce code duplication a little.

**Review by josokinas — APPROVED** [review]

File: src/locales/en_US/en_US.json [hunk 2]
```diff
@@ -16,11 +16,18 @@
             "last_name": "Last name",
```
Comment (osolliec) [inline src/locales/en_US/en_US.json:16]: could you please add strings for all languages at the same time please ?

**Review by osolliec — CHANGES_REQUESTED** [review]

File: src/locales/en_US/en_US.json (hunk 2 above)
Comment (acMaiaOnf) [reply src/locales/en_US/en_US.json:16]: I don't have access to lokalise and I don't have the all the strings. I already asked for them, so I'll add them as soon as I can.

Comment (osolliec) [reply src/locales/en_US/en_US.json:16]: there's a unit test in the dev branch that will fail if the locales are not in sync with the en_US file, just fyi :)

**Review by josokinas — APPROVED** [review]

**Comment by josokinas** [comment]
One thing to note, that probably this would need to still support old way of providing SSN configuration, for legacy workflows.

**Comment by acMaiaOnf** [comment]
It does @josokinas 
Currently it's passing the `ssn_enabled` flag with

`
      ssnEnabled={
        dataStep?.options?.ssn_enabled ||
        dataStep?.options?.profile_data_selection?.ssn_enabled
      }
`

**Comment by acMaiaOnf** [comment]
And also for the other screens it's checking if it has the enabled flag or it's `true` or `false`

**Review by osolliec — APPROVED** [review]
would it be possible to minimize the diffs from the locales.json files ? just to ease the reviews regarding translations (maybe we should auto format them in prettierrc file, not sure).

**Pull request merged**